### DISCUSSION TITLE
Error: Getting metadata from plugin failed with error: invalid_grant

### DISCUSSION THREAD

**Discussion by dahcucumber** [Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client#372] [Q&A]
I tried to restart the program, uninstall and reinstall and it doesn't work, what can i do about that? Edit: FAQ answer doesn't help
![Zrzut ekranu 2021-05-11 094833](https://user-images.githubusercontent.com/83413986/117781150-9d33c980-b240-11eb-98b0-a30d16a68f17.png)

**Comment by Melvin-Abraham** [comment]
Please refer this section of [Frequently Asked Questions (FAQ)](https://github.com/Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client/wiki/Frequently-Asked-Questions-(FAQ)#how-do-i-fix-invalid_grant-error)

**Reply by dahcucumber** [reply]
It doesn't help still

**Reply by twelvemp** [reply]
I have the same problem, I tried what it said to do in the FAQs and a new token is not created.

**Reply by Melvin-Abraham** [reply]
Make sure the "Saved Token Path" does not point to locations requiring admin or superuser privileges like if you are in windows make sure the Token path is not within `C:`...
